packages/grafana-data/src/types/suggestions.ts

```diff
@@ -0,0 +1,8 @@
+import { DataFrame, PanelPlugin, VariableModel, VariableSuggestion, VariableSuggestionsScope } from '..';
+
+export type SuggestionsProvider = (
+  plugin: PanelPlugin,
```

Comment:
do we need PanelPlugin?  it would be nice to have the same interface for DataSource query editors

Reply:
nope, we can rely on data being provided or not IMO